pkg/config/log.go

```diff
@@ -49,7 +60,7 @@ func getSyslogTLSKeyPair() (*tls.Certificate, error) {
 }
 
 // SetupLogger sets up the default logger
-func SetupLogger(logLevel, logFile, uri string, rfc, logToConsole, jsonFormat bool) error {
+func SetupLogger(loggerName LoggerName, logLevel, logFile, uri string, rfc, logToConsole, jsonFormat bool) error {
```

Comment:
Since other agents will want to use this now, it would be nice to document this function thoroughly, explaining what each parameter is. For example, to me it isn't clear what `uri`, `rfc` and `logToConsole` are. `loggerName` is also very vague. How is it used? What does it mean?

Reply:
Agreed, added a better documentation on the function